contrib/gn_module_validation/backend/gn_module_validation/blueprint.py

```diff
@@ -69,7 +69,8 @@ def get_synthese_data(info_role):
         'taxref.nom_vern',
         'taxref.lb_nom',
         'taxref.nom_vern_or_lb_nom',
-        'dataset.validable'
+        'dataset.validable',
+        'reports.report_type.type'
```

Comment:
Il ne faut pas rajouter ``reports`` à ``fields`` ici car la suite du code va alors joindre ``reports``, ce qui pose des problèmes de performance s’agissant d’une relation one-to-many et donc dupliquant les info de la synthèse dans la réponse SQL. Il faut donc le rajouter à la fin de la fonction, juste avant l’appelle à ``as_geofeaturecollection``.